connectivity/drivers/emac/TARGET_NXP_EMAC/TARGET_IMX/CMakeLists.txt

```diff
@@ -1,14 +1,16 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-os-emac-imx INTERFACE)
+
 mbed_add_cmake_directory_if_labels("TARGET")
```

Comment:
Can you explain why you did 
```
if("CORDIO_LL" IN_LIST MBED_TARGET_LABELS)
    add_subdirectory(TARGET_CORDIO_LL)
endif()
```
but in many other places you are still using `mbed_add_cmake_directory_if_labels()`

Note: I am posting this comment multiple time to keep a record where to modify in case you have to.